### ISSUE TITLE
Lancedb serverless.yml

### ISSUE THREAD

**Issue by jinman** [lancedb/lancedb#203]
I really liked the tutorial https://lancedb.github.io/lancedb/examples/serverless_lancedb_with_s3_and_lambda/. Is there a serverless.yml (serverless framework) to getting started that could make this super simple as all the plumbing might be configured. Thoughts?

**Comment by jaichopra** [comment]
This is a great idea. I think we can have a yml + templated docker file containing all dependencies/lambda code.

Do let me know if you'd like to contribute and/or collaborate on this

**Comment by jinman** [comment]
Yes we will absolutely love to contribute and help test. But we are a bit new to vector dbs, in general. Dumb question: Do you need a docker file or can you install all the dependencies in as a lambda layer?

**Comment by LincolnCannon** [comment]
Anyone have any luck creating a serverless layer that works with nodejs? I'm getting "libgfortran.so.5: cannot open shared object file: No such file or directory" when I try to require "vectordb".

**Comment by poupalov** [comment]
@jinman @jaichopra @LincolnCannon if it's still helpful, I made a demo of deploying a Lambda function which uses `vectordb` with the Serverless framework and `esbuild`: [vectordb-js-lambda](https://github.com/poupalov/vectordb-js-lambda).

The key to making it work was:
- creating a Lambda layer to provide the right Linux binary to my Lambda function (i.e. @lancedb/vectordb-linux-x64-gnu)
- marking vectordb as an external dependency when bundling my Lambda function with esbuild
